desktop/apps/categories2/components/Sidebar.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react'
 
-export default class Sidebar extends React.Component {
-  render(){
-    const { data } = this.props
```

Comment:
@oxaudo Here's some more about that https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/